charts/sptribs-frontend/values.yaml

```diff
@@ -16,7 +16,7 @@ nodejs:
     IDAM_API_URL: 'https://idam-api.{{ .Values.global.environment }}.platform.hmcts.net/o/token'
     REDIS_HOST: 'sptribs-{{ .Values.global.environment }}.redis.cache.windows.net'
     EQUALITY_URL: 'https://pcq.{{ .Values.global.environment }}.platform.hmcts.net'
-    CASE_API_URL: 'http://sptribs-case-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal'
+    CASE_API_URL: 'https://sptribs-case-api-pr-1810.preview.platform.hmcts.net'
```

Comment:
We just need to remember to revert this change while releasing the code changes.

Reply:
Yeah will do, leaving it there for testing

Reply:
Does this still need to be kept for testing?